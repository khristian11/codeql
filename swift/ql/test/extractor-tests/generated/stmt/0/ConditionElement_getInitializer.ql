// generated by codegen/codegen.py
import codeql.swift.elements
import TestUtils

from ConditionElement x
where toBeTested(x)
select x, x.getInitializer()
