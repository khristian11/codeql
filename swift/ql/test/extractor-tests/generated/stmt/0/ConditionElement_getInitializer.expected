| statements.swift:67:4:67:19 | let ... = ...  | statements.swift:67:19:67:19 | x |
| statements.swift:69:4:69:20 | .some(...) = ...  | statements.swift:69:20:69:20 | x |
